libs/awscdk/src/api.ts

```diff
@@ -229,10 +237,10 @@ export class Api extends cloud.Api implements IAwsApi {
   }
 
   /** @internal */
-  public _toInflight(): string {
-    return core.InflightClient.for(__dirname, __filename, "ApiClient", [
-      `process.env["${this.urlEnvName()}"]`,
-    ]);
+  public _liftedState(): Record<string, string> {
+    return {
+      $url: `process.env["${this.urlEnvName()}"]`,
```

Comment:
Why do we need the $ here?

Reply:
Hmm, yeah in retrospect the dollar signs aren't necessary. I think I did this originally just to mentally match up the output between what the compiler's generating for Wing source code and what's in the SDK while I was designing the solution, but in practice naming doesn't really matter.